packages/cli/src/zosfiles/compare/CompareBaseHelper.ts

```diff
@@ -171,28 +171,40 @@ export class CompareBaseHelper {
     }
 
     /**
-     * To get the difference string in ternninal or in browser
-     * @param {string} string1 - string of file 1 comtent
-     * @param  {string} string2 - string of file 2 comtent
+     * To get the difference string in terminal or in browser
+     * @param {string} string1 - string of file 1 content
+     * @param {string} string2 - string of file 2 content
+     * @param {IDiffOptions} options
      * @returns {IZosFilesResponse}
      * @public
      * @memberof CompareBaseHelper
      */
-    public async getResponse(string1: string, string2: string): Promise<IZosFilesResponse>{
-        //  CHECHKING IIF THE BROWSER VIEW IS TRUE, OPEN UP THE DIFFS IN BROWSER
+    public async getResponse(string1: string, string2: string, options?: IDiffOptions): Promise<IZosFilesResponse>{
```

Comment:
This is more of a style choice and this will come off as nitpicking, but I don't find the parameter naming convention to be clear here. Within the context of a diff, I think someone can infer what `string1` and `string2` are, but self-describing parameters would help clear up any confusion for others that might incorporate/use this compare class.

Maybe something like `oldContent` and `newContent` would work here? If you want to maintain consistency with `IDiffOptions`, then feel free to disregard this comment, just a suggestion 😅 This could be something to consider down the road